webapp/src/app.tsx

```diff
@@ -80,19 +80,24 @@ const App = React.memo((): JSX.Element => {
     const globalError = useAppSelector<string>(getGlobalError)
     const me = useAppSelector<IUser|null>(getMe)
     const dispatch = useAppDispatch()
+    // this is used to detect if we're running inside the plugin but
```

Comment:
Can we explain here that this is a temporary solution until we find a better way to provide annonymous/not-websocket connected routes?